nengo_gui/components/ace_editor.py

```diff
@@ -6,18 +6,18 @@
 import nengo_gui.monkey
 
 class AceEditor(Component):
-    def __init__(self, viz, config, uid):
-        super(AceEditor, self).__init__(viz, config, uid)
-        self.viz = viz
+    def __init__(self, sim, config, uid):
+        super(AceEditor, self).__init__(sim, config, uid, z_order=-8)
```

Comment:
Good point; I should document that better.  Basically, in the old version there was code scattered all over the place that ensured that SimControl was first in the components list, followed by AceEditor and NetGraph, followed by everything else.  This consolidates all of that into one sort.  

Old code:

https://github.com/nengo/nengo_gui/blob/master/nengo_gui/viz.py#L76
https://github.com/nengo/nengo_gui/blob/master/nengo_gui/viz.py#L140
https://github.com/nengo/nengo_gui/blob/master/nengo_gui/components/netgraph.py#L211